src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -34,7 +34,7 @@ curie_map:
   semapv: https://w3id.org/semapv/vocab/
   rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
   sssom: https://w3id.org/sssom/
-  oio: http://www.geneontology.org/formats/oboInOwl#
+#  oio: http://www.geneontology.org/formats/oboInOwl#
```

Comment:
## Partial or merged EPM?
@cthoyt Right now we only have `oio` and `oboInOwl` in conflict. I can think of several ways to address this. I could convert our entire flat prefix map into an EPM, and I may well do that. But I wonder about your thoughts. Consider:


Plain prefix map:
```yaml
  MONDO: http://purl.obolibrary.org/obo/MONDO_
  UMLS: http://linkedlifedata.com/resource/umls/id/
  NCIT: http://purl.obolibrary.org/obo/NCIT_
  DOID: http://purl.obolibrary.org/obo/DOID_
```

EPM:
```python
{
   'prefix': 'oboInOwl',
   'prefix_synonyms': [],
   'uri_prefix': 'http://www.geneontology.org/formats/oboInOwl#',
   'uri_prefix_synonyms': []
}
```

Let's say I wanted to merge the two programmatically. What do you think would be the best option:
a. A new complex data structure that allows inclusion of both a plain flat prefix map as well as an EPM
b. Keep these two structures separate, but use some code that merges them together, e.g.
  b.1. by converting the flat prefix map to an EPM and merges that
  b.2. a method that merges 2 `Converter` objects (`curies.chain()`?)
c. something else?

Reply:
I would highly suggest converting the flat prefix map into an EPM as a first step and have a single source of truth. You're asking good questions that all boil down to: how do I manage my prefix map, given there are so many different streams from which definitions of CURIE prefix-URI prefix pairs come from?


This problem is so complicated that we built a solution into the Bioregistry https://bioregistry.io/context. My preferred solution is to bake in the Monarch community rules into a "context" definition, then all of the merging and reconciliation get taken care of by the Bioregistry and give you an EPM. @matentzn has already done this to make an "OBO" context.

Reply:
Just as an aside this here has nothing at all to do with any "Monarch" prefix maps - this is all ETL, build long before curies and EPMs; I agree with constructing an EPM, if it is necessary at all - I just want to make sure why understand the problem and _that_ it is necessary, or we can get away with simply deleting the conflicting entires in the prefix maps.

Reply:
Thanks for getting back, @cthoyt.

@matentzn I explained a few times in https://github.com/mapping-commons/sssom-py/pull/485 that deleting the conflicting entry doesn't work. A prefix map is getting dynamically generated based on what OAK is interrogating from Mondo. Even if I were to delete `oio` or `oboInOwl` after it was dynamically generated, it will throw an error later because it can't resolve.